.scalafix.conf

```diff
@@ -1,5 +1,5 @@
 rules = [
-  ExplicitResultTypes,
+#  ExplicitResultTypes,
```

Comment:
temporarily disable that rule: despite [forcing the latest version of scalameta](https://github.com/scalacenter/scalafix/blob/b3752b387349c0ce07840a6394647a118fd4ed21/build.sbt#L12) for the run of scalafix in this repo, sbt-scalafix runs with the current version of scalafix which [does not whitelist 2.13.8 yet](https://github.com/scalacenter/scalafix/blob/8e64923d1a22a4dc741f278a4fc0339c4fca1e92/scalafix-rules/src/main/scala/scalafix/internal/rule/ExplicitResultTypes.scala#L31)